programs/loader-v4/Cargo.toml

```diff
@@ -40,6 +40,7 @@ name = "solana_loader_v4_program"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [features]
+agave-internal = []
```

Comment:
i would prefer that the feature flag be more severe in it's naming

my vote would be for `agave-unstable-api`

Reply:
Yeah, that's very reasonable. I'll update the PR.